ee/codegen_integration/fixtures/simple_scheduled_trigger/display_data/simple_scheduled_trigger.json

```diff
@@ -168,7 +168,32 @@
       "type": "SCHEDULED",
       "cron": "* * * * *",
       "timezone": "America/New_York",
-      "attributes": [],
+      "attributes": [
```

Comment:
[oos] I wouldn't expect to need these serialized, since we get them for free from the ScheduledTrigger base class